litestar/openapi/plugins.py

```diff
@@ -262,7 +262,7 @@ def create_request_interceptor(csrf_config: CSRFConfig) -> str:
         """
 
         if request.app.csrf_config:
-            c = request.app.csrf_config
+            c = request.app.csrf_config  # pragma: no cover
```

Comment:
Why no coverage for these?

Reply:
we dont use `Litestar(csrf_config=...)` anymore in the tests but I can tweak them so that it's using that branch

Reply:
Mhm. As long as it's just deprecated, it should still be supported and therefore tested

Reply:
ok modified